### PR TITLE
[Fix] Move form subtitle into separate component and styling clean up

### PR DESCRIPTION
**Pull request by dynamic11**
#### Changes introduced

⭐ New Features
- moved form sub title into separate components

🩹 Fixed Features
- clean up form title styling
- removed unused styling code from `profile form` components
- brought back missing "visibility options" in form title during profile setup

#### Related issue(s)
✔️ related to code review received in #1013

### CODE REVIEW

File: services/frontend/src/components/profileForms/formTitle/FormTitleView.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  margin-left: 10px;
+  font-weight: normal;
+  font-style: italic;
+  opacity: 0.5;
+}
+
+.profileForm-extra {
+  display: inline;
+  float: right;
+}
```
Comment (BenJeau) [inline services/frontend/src/components/profileForms/formTitle/FormTitleView.scss]: Looks good, small comment I think we should slowly be moving to this form for our scss files to address #995

```suggestion
.profileForm {
  &-title {
    font-size: 1.2em !important;
    margin: 0;
  }

  &-unsavedText {
    font-size: 0.9em !important;
    margin-left: 10px;
    font-weight: normal;
    font-style: italic;
    opacity: 0.5;
  }

  &-extra {
    display: inline;
    float: right;
  }
}
```

File: services/frontend/src/components/profileForms/careerManagementForm/CareerManagementFormView.scss [hunk 2]
```diff
@@ -4,14 +4,9 @@
   min-height: 343px;
   padding: 30px 30px;
 }
-.pgf-formTitle {
```
Comment (Mousto097) [inline services/frontend/src/components/profileForms/careerManagementForm/CareerManagementFormView.scss]: @dynamic11 The ".pgf-formTitle" class is being used at 2 others places in CareerManagementFormView.jsx, but it was removed from the CareerManagementFormView.scss. Is this something we need to fix?

See screenshot

Screen Shot 2020-11-13 at 2 24 13 PM

**Review by Mousto097 — CHANGES_REQUESTED** [review]

File: services/frontend/src/components/profileForms/formTitle/FormTitleView.scss (hunk 1 above)
Comment (dynamic11) [reply services/frontend/src/components/profileForms/formTitle/FormTitleView.scss]: Thank you for the reminder. Will do!

**Comment by Mousto097** [comment]
I think we should also get rid of this class "qual-unsavedText" in QualificationsFormView.scss. It is not being used anywhere!

Screen Shot 2020-11-13 at 2 42 25 PM

File: services/frontend/src/components/profileForms/careerManagementForm/CareerManagementFormView.scss (hunk 2 above)
Comment (dynamic11) [reply services/frontend/src/components/profileForms/careerManagementForm/CareerManagementFormView.scss]: Yes! I will look into it

**Comment by dynamic11** [comment]
> I think we should also get rid of this class "qual-unsavedText" in QualificationsFormView.scss. It is not being used anywhere!
> 
> 
> 
> Screen Shot 2020-11-13 at 2 42 25 PM
> 
> 

Yea I think that exists in every form. I should clean it all out

**Review by Mousto097 — CHANGES_REQUESTED** [review]
The  section header title font-size are inconsistent (see screenshots)

Screen Shot 2020-11-13 at 2 57 51 PM
Screen Shot 2020-11-13 at 2 58 35 PM

**Comment by BenJeau** [comment]
Seems to be missing the visibility toggles in the top header of these two forms (only in the create form flow tho)

image
image

**Comment by dynamic11** [comment]
@BenJeau it doesn't seem to be there in dev either but ill investigate further

**Comment by BenJeau** [comment]
Oh you're right, could you fix it in this branch since you're touching most of the forms? I think you just need to add them to the Section header like when the location does not contain the create string

**Comment by dynamic11** [comment]
found the issue. will push the fix in a few minutes

**Comment by dynamic11** [comment]
fixed!

**Review by BenJeau — APPROVED** [review]

**Review by Mousto097 — APPROVED** [review]

**Pull request merged**